.vib/kube-prometheus/cypress/cypress/integration/kube_prometheus_spec.js

```diff
@@ -7,7 +7,7 @@ it('allows executing a query and displaying response data for each deployment',
   Object.keys(deployments).forEach((podName, i) => {
     const query = Object.values(deployments)[i].query;
 
-    cy.get('[role="textbox"]').clear().type(query);
+    cy.get('[role="textbox"]').clear().type(`${query}{enter}`);
     cy.contains('Execute').click({force: true});
```

Comment:
Now that we have ensured the origin of the banner covering the element and applied measures to prevent it from appearing (forced `clear()` and used `enter`), there is no need to use the `force` here.

```suggestion
    cy.contains('Execute').click();
```